cpp/src/arrow/flight/sql/server_test.cc

```diff
@@ -381,6 +382,49 @@ TEST_F(TestFlightSqlServer, TestCommandGetTablesWithIncludedSchemas) {
   AssertTablesEqual(*expected_table, *table);
 }
 
+TEST_F(TestFlightSqlServer, TestCommandGetTypeInfo) {
+  ASSERT_OK_AND_ASSIGN(auto flight_info, sql_client->GetTypeInfo({}));
+
+  ASSERT_OK_AND_ASSIGN(auto stream,
+                       sql_client->DoGet({}, flight_info->endpoints()[0].ticket));
+
+  const std::shared_ptr<Schema>& expected_schema = SqlSchema::GetTypeInfoSchema();
```

Comment:
nit, but schema/batch in these tests aren't references - they can just be declared `const std::shared_ptr<Schema>` or just `auto`.

Reply:
Fixed